diff -up libksba-1.0.7/src/ksba-config.in.multilib libksba-1.0.7/src/ksba-config.in
--- libksba-1.0.7/src/ksba-config.in.multilib	2006-08-31 10:05:24.000000000 +0200
+++ libksba-1.0.7/src/ksba-config.in	2013-05-07 16:14:21.258066067 +0200
@@ -116,9 +116,6 @@ fi
 
 if test "$echo_libs" = "yes"; then
     libdirs=""
-    if test "@libdir@" != "/usr/lib" ; then
-      libdirs="-L@libdir@"
-    fi
     tmp=""
     for i in $libdirs $libs; do
        if echo "$tmp" | fgrep -v -- "$i" >/dev/null; then
